app/src/main/java/com/example/localieapp/adapter/GridAdapter.kt

```diff
@@ -62,5 +64,15 @@ class GridAdapter(private val context: Context, private val dataset: List<Coupon
                 .diskCacheStrategy(DiskCacheStrategy.DATA)
                 .into(holder.imageView)
         }
+        holder.itemView.setOnClickListener(View.OnClickListener() {
+            Log.d("onBindViewHolder:", position.toString())
+        })
+
```

Comment:
Why do we have to do this twice? Should we only have to make this change in the EarnGripAdapter?